Foundation/NSOrderedSet.swift

```diff
@@ -420,10 +428,6 @@ extension NSMutableOrderedSet {
     }
     
     open func exchangeObject(at idx1: Int, withObjectAt idx2: Int) {
-        guard idx1 < count && idx1 >= 0 && idx2 < count && idx2 >= 0 else {
```

Comment:
These checks are now performed by the calls to object(at:) on lines 431 and 432.